features/admin/promotion/removing_promotions/preventing_removal_of_promotions_in_use.feature

```diff
@@ -16,7 +16,7 @@ Feature: Prevent deletion of promotions applied to order
         And the customer chose "Free" shipping method to "United States" with "Cash on Delivery" payment
         And I am logged in as an administrator
 
-    @api @ui
+    @api-todo @ui
```

Comment:
I couldn't cover these scenarios because the `GetCollection` endpoint is not yet available.